x/bank/autocli.go

```diff
@@ -87,5 +87,34 @@ func (am AppModule) AutoCLIOptions() *autocliv1.ModuleOptions {
 				},
 			},
 		},
+		Tx: &autocliv1.ServiceCommandDescriptor{
+			Service:              bankv1beta1.Msg_ServiceDesc.ServiceName,
+			EnhanceCustomCommand: true,
+			RpcCommandOptions: []*autocliv1.RpcCommandOptions{
+				{
+					RpcMethod: "Send",
+					Use:       "send [from_key_or_address] [to_address] [amount]",
+					Short:     "Send funds from one account to another.",
+					Long: `Send funds from one account to another.
+			Note, the '--from' flag is ignored as it is implied from [from_key_or_address].
+			When using '--dry-run' a key name cannot be used, only a bech32 address.`,
+					PositionalArgs: []*autocliv1.PositionalArgDescriptor{{ProtoField: "to_address"}, {ProtoField: "amount"}},
+				},
+				{
+					RpcMethod:      "Burn",
+					Use:            "burn [from_key_or_address] [amount]",
+					Short:          "Burns the amount specified from the given account.",
+					PositionalArgs: []*autocliv1.PositionalArgDescriptor{{ProtoField: "amount"}},
```

Comment:
This does not match the usage description. If you want the `from_key_or_address` as a positional argument, you need to add it here.